src/style.css

```diff
@@ -28,6 +29,16 @@ button {
   cursor: pointer;
   width: fit-content;
   padding: 5px 8px;
+  transition: all 250ms ease-in-out;
+  background: #fff;
+  border: 1px solid rgb(28, 35, 62);
+  border-radius: 3px;
+}
+
+button:hover,
+#submit:hover {
+  padding: 7px 11px;
+  cursor: pointer;
 }
```

Comment:
- [ ] In your readme file kindly add the **Usage** section and illustrate how to use the app.

- [ ] In your readme headers kindly remove the following links: Live Demo, Run tests, and Deployment as they are broken links and have no related sections.

- [ ] Your app layout should match the design. In your case, the elements should be centered on the page, but your elements are aligned to the left side of the page. Kindly fix that.

  Your Implementation | Wireframe Design
  :---:|:---:
  ![nfjiw](https://user-images.githubusercontent.com/81468675/225317473-682bbedf-15de-4820-a480-0b2a5f280ec6.jpg) | ![rirrp](https://user-images.githubusercontent.com/81468675/225317485-eb344eb1-d3dc-4f42-8f94-ed3c5fe2bda3.jpg)